components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/handlers/HybridResponseTypeHandler.java

```diff
@@ -55,20 +55,21 @@ public OAuth2AuthorizeRespDTO issue(OAuthAuthzReqMessageContext oauthAuthzMsgCtx
             ResponseTypeHandlerUtil.buildAuthorizationCodeResponseDTO(respDTO, authzCodeDO);
         }
 
-        // Generating access token and generating response for access token flow.
-        if (isAccessTokenIssued(responseType)) {
+        // Generating a single access token if id_token and/or token is in response_type
```

Comment:
Let's format this by adding a "." at the end.

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1444